### PR TITLE
[Minor] Remove dead code & fix phpdoc

### PR DESCRIPTION
**Pull request by SofHad**

### CODE REVIEW

File: src/Silex/ServiceProviderInterface.php [hunk 1]
```diff
@@ -34,6 +34,8 @@ public function register(Application $app);
      * This method is called after all services are registered
      * and should be used for "dynamic" configuration (whenever
      * a service must be requested).
+     *
+     * @param Application $app An Application instance
```
Comment (fabpot) [inline src/Silex/ServiceProviderInterface.php]: This should be removed as it does not add any value over the signature.

**Comment by SofHad** [comment]
I removed it for register too. To harmonize both methods ?

File: src/Silex/ServiceProviderInterface.php [hunk 2]
```diff
@@ -23,8 +23,6 @@
      *
      * This method should only be used to configure services and parameters.
      * It should not get services.
-     *
-     * @param Application $app An Application instance
```
Comment (henrikbjorn) [inline src/Silex/ServiceProviderInterface.php]: This needs to stay, also it should be added to the boot method.

Comment (fabpot) [inline src/Silex/ServiceProviderInterface.php]: No, it does not add any value, the method signature already tell you the same.

**Comment by fabpot** [comment]
Thank you @SofHad.

File: src/Silex/ServiceProviderInterface.php (hunk 2 above)
Comment (henrikbjorn) [inline src/Silex/ServiceProviderInterface.php]: Isn' the phpdoc invalid if it contains information, but not the parameters used? I am not saying the "An application instance" should stay. Just the `@param` tag.

**Pull request closed without merging**